### ISSUE TITLE
Edges when displaying atom positions and blockade radius

### ISSUE THREAD

**Issue by jon-wurtz** [QuEraComputing/bloqade-python#693]
**Is your feature request related to a problem? Please describe.**
When plotting atom positions, it is possible to show the unit disk radius but it is not necessarily clear which vertices are connected within the blockade radius

**Describe the solution you'd like**
Add lines between vertices which are within the blockade radius. For example, consider the plot
![image](https://github.com/QuEraComputing/bloqade-python/assets/20091330/5a134127-bdd9-44de-86bc-cdb11bf25215)

**Comment by weinbe58** [comment]
@kaihsin Do you think this is possible?

**Comment by kaihsin** [comment]
Yes this is actually something I would do to improve next. Sorry I didn't being active on here.   

There is an example on the bokeh gallery for this, and it's definitely possible but need to do some preprocessing of the data (and generate corresponding widgets)

**Comment by weinbe58** [comment]
No worries! There will be time to do this :)
